public/login.html

```
@@ -8,140 +9,177 @@
-        </div>
-        <p id="login-err" class="text-red-500 text-sm hidden"></p>
-        <button type="submit" class="w-full bg-brand text-white font-semibold py-2.5 rounded-xl hover:bg-brand-dark transition text-sm">Sign In</button>
-      </form>
-      <p class="text-center text-slate-400 text-sm mt-6">
-        Demo: <code class="bg-slate-100 px-1 rounded">alice / password123</code>
-        or <code class="bg-slate-100 px-1 rounded">charlie / password123</code>
-      </p>
-    </div>
+    <div class="bg-white rounded-2xl shadow-sm border border-slate-100 overflow-hidden">
+      <div class="flex border-b border-slate-100">
+        <button id="tab-login" onclick="switchTab('login')"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)